dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+            params: {
+                placement: "2222"
+            }
+       }
+   ]
+}]
+
+{% endhighlight %}
+
+## Use Case: What if some bid requests apply only to users originating certain countries? 
+
+Labels aren't constrained to describing device size -- they can be used for any condition the page wants
+to define. To support this, labels can now be passed into the `requestBids()` function as an argument.
+
+A specific use case: suppose that a certain bidder doesn't have a data center outside of a
+certain region. It's really not worth sending them bid
+requests for users outside of their geographic area. Assuming the page can figure out where the user's from,
+a label can be implemented and applied to make the bid conditional.
+
+{% highlight js %}
+// page logic determines the 'europeanUser' boolean
+If (europeanUser) {
+    reqArgs={labels:['eur']};
+} 
+requestBids(reqArgs);
+{% endhighlight %}
+
+Then this label can be applied to conditions in the AdUnit just like labels that originate from sizeConfig. E.g.
+
+{% highlight js %}
+var AdUnits = [{
+    code: "ad-slot-1",
+    sizes: [[768,90], [468,60], [320,50]], 
+    bids: [
+       {
+            bidder: "euroMobileBidder",
+            labelAll: ["eur", "phone"],
+            params: {
+                placement: "9876"
+            }
+       },
+       ...
+   ]
+}]
+{% endhighlight %}
+
+This example shows that the 'euroMobileBidder' is only interested in receiving bids that have **both**
+labels: both "eur" as passed into requestBids and "phone" as created by sizeConfig.
+
+
+## Syntax Notes
+
+Probably you've gotten the gist of the feature by now, but it's still worth laying out the syntax a little more. In Prebid 1.0, label targeting is constrained to two relatively simple operators: `labelAny` and `labelAll`.
+
+With the `labelAny` conditional, just one label has to match for the condition to be true.
+{% highlight bash %}
+labelAny: ["A", "B"]
+// Either A or B can be defined in the label array to activate this bid or AdUnit
+{% endhighlight %}
+
+With the `labelAll` conditional, Every element of the target array must match an element of the label array in order for the condition to be true.
+{% highlight bash %}
+labelAll: ["A", "B"] 
+// both A and B must be defined in the label array to activate this bid or AdUnit
+{% endhighlight %}
+
+
+{: .alert.alert-warning :}
+**This feature only works in Prebid 1.0 and later.**
+Only one conditional may be specified on a given AdUnit or bid -- if both labelAny and labelAll are specified, only the first one will be utilized and an error will be logged to the console. It is allowable for an AdUnit to have one condition and a bid to have another.
+
+We may someday implement full boolean logic in this feature, but the common use cases don't demand the complexity or code size at this point.
+
+
+## Further Reading
+
++ [Responsive ad designs]({{site.baseurl}}/dev-docs/feature-responsive.html)
```

Comment:
I think this should be a link to [the API reference's explanation of using labels to show responsive ads](http://prebid.org/dev-docs/publisher-api-reference.html#setConfig-Configure-Responsive-Ads)